test/performance/odp_timer_accuracy.c

```diff
@@ -1008,6 +1010,37 @@ static void print_stat(test_global_t *test_global)
 	print_nsec_error("max", max, res_ns, -1, -1);
 
 	printf("\n");
+
+	if (test_global->common_options.is_export) {
+		if (test_common_write("num after,num before,num exact,num retry,"
+				      "error after min (nsec),error after min resolution,"
+				      "error after max (nsec),error after max resolution,"
+				      "error after ave (nsec),error after ave resolution,"
+				      "error before min (nsec),error before min resolution,"
+				      "error before max (nsec),error before max resolution,"
+				      "error before ave (nsec),error before ave resolution,"
+				      "final timeout error max (nsec),"
+				      "final timeout error max resolution\n"))
+			goto exit;
+
+		if (test_common_write("%i,%i,%i,%i,%i,%f,%i,%f,%i,%f,%i,%f,%i,%f,%i,%f,%i,%f\n",
+				      stat->num_after, stat->num_before,
+				      stat->num_exact, stat->num_too_near,
+				      stat->nsec_after_min, (double)stat->nsec_after_min / res_ns,
+				      stat->nsec_after_max, (double)stat->nsec_after_max / res_ns,
+				      ave_after, (double)ave_after / res_ns,
+				      stat->nsec_before_min, (double)stat->nsec_before_min / res_ns,
+				      stat->nsec_before_max, (double)stat->nsec_before_max / res_ns,
+				      ave_before, (double)ave_before / res_ns,
+				      max, (double)max / res_ns
+				      ))
+			goto exit;
```

Comment:
Failures to export test results are not detected.

Reply:
`print_stat()` needs still a return value, so an export failure could be easily detected for example in CI.